rest-client/src/test/java/org/sagebionetworks/bridge/sdk/rest/AuthenticationHandlerIntegrationTest.java

```diff
@@ -0,0 +1,90 @@
+package org.sagebionetworks.bridge.sdk.rest;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.sagebionetworks.bridge.sdk.Config;
+import org.sagebionetworks.bridge.sdk.rest.api.AuthenticationApi;
+import org.sagebionetworks.bridge.sdk.rest.api.UploadsApi;
+import org.sagebionetworks.bridge.sdk.rest.model.SignIn;
+import org.sagebionetworks.bridge.sdk.rest.model.UploadRequest;
+
+import java.util.Properties;
+
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+
+/**
+ * Created by liujoshua on 9/8/16.
+ */
+public class AuthenticationHandlerIntegrationTest {
+  private static final String STUDY = "api";
+  private static final String BASE_URL = "https://webservices-develop.sagebridge.org/";
+
+  private AuthenticationHandler authenticationHandler;
+  private Retrofit bridgeAuthenticatedRetrofit;
+
+  @Before
+  public void before() {
+    Retrofit retrofit = new Retrofit.Builder().baseUrl(BASE_URL)
+                                              .addConverterFactory(GsonConverterFactory.create())
+                                              .build();
+
+    Properties properties = Config.instance.properties;
+
+    String adminEmail = properties.getProperty(Config.Props.ADMIN_EMAIL.getPropertyName());
+    String adminPassword = properties.getProperty(Config.Props.ADMIN_PASSWORD.getPropertyName());
+
+    SignIn credentials = new SignIn().study(STUDY).email(adminEmail).password(adminPassword);
+    AuthenticationApi authenticationService = retrofit.create(AuthenticationApi.class);
+
+
+    authenticationHandler = new AuthenticationHandler(credentials,
+                                                      new UserSessionInfoProvider(
+                                                          authenticationService)
+    );
+
+    // AuthenticationHandler auth must be registered as both an interceptor and an authenticator
+    OkHttpClient authedHttpClient = new OkHttpClient.Builder().authenticator(authenticationHandler)
+                                                              .addInterceptor(authenticationHandler)
+                                                              .build();
```

Comment:
Ok, I changed my formatter to wrap instead of chop down, and not to align the method calls